src/model/test/ZoneMixing_GTest.cpp

```diff
@@ -205,3 +214,268 @@ TEST_F(ModelFixture, ZoneMixing_ZoneRemove) {
+  ASSERT_TRUE(zmPerson.flowRateperPerson());
+  EXPECT_EQ(m3sPerPerson, zmPerson.flowRateperPerson().get());
+  EXPECT_FALSE(zmPerson.airChangesperHour());
+  ASSERT_TRUE(zmPerson.space());
+  EXPECT_EQ(space4, zmPerson.space().get());
+  EXPECT_FALSE(zmPerson.zone());
+  ASSERT_TRUE(zmPerson.sourceSpace());
+  EXPECT_EQ(space12_.get(), zmPerson.sourceSpace().get());
+  EXPECT_FALSE(zmPerson.sourceZone());
+
+  ASSERT_TRUE(zmACH.designFlowRate());
+  EXPECT_EQ(equivm3s_ACH, zmACH.designFlowRate().get());
+  EXPECT_FALSE(zmACH.flowRateperFloorArea());
+  EXPECT_FALSE(zmACH.flowRateperPerson());
+  EXPECT_FALSE(zmACH.airChangesperHour());
+  ASSERT_TRUE(zmACH.space());
+  EXPECT_EQ(space12_.get(), zmACH.space().get());
+  EXPECT_FALSE(zmACH.zone());
+  ASSERT_TRUE(zmACH.sourceSpace());
+  EXPECT_EQ(space3, zmACH.sourceSpace().get());
+  EXPECT_FALSE(zmACH.sourceZone());
+
+  auto space34_ = z2.combineSpaces();
+  ASSERT_TRUE(space34_);
+
+  EXPECT_EQ(2, m.getConcreteModelObjects<Space>().size());
+
+  ASSERT_TRUE(zmAbs.designFlowRate());
+  EXPECT_EQ(m3sAbs, zmAbs.designFlowRate().get());
+  EXPECT_FALSE(zmAbs.flowRateperFloorArea());
+  EXPECT_FALSE(zmAbs.flowRateperPerson());
+  EXPECT_FALSE(zmAbs.airChangesperHour());
+  ASSERT_TRUE(zmAbs.space());
+  EXPECT_EQ(space34_.get(), zmAbs.space().get());
+  EXPECT_FALSE(zmAbs.zone());
+  ASSERT_TRUE(zmAbs.sourceSpace());
+  EXPECT_EQ(space12_.get(), zmAbs.sourceSpace().get());
+  EXPECT_FALSE(zmAbs.sourceZone());
+
+  ASSERT_TRUE(zmFloor.designFlowRate());
+  EXPECT_EQ(equivm3s_Floor, zmFloor.designFlowRate().get());
+  EXPECT_FALSE(zmFloor.flowRateperFloorArea());
+  EXPECT_FALSE(zmFloor.flowRateperPerson());
+  EXPECT_FALSE(zmFloor.airChangesperHour());
+  ASSERT_TRUE(zmFloor.space());
+  EXPECT_EQ(space12_.get(), zmFloor.space().get());
+  EXPECT_FALSE(zmFloor.zone());
+  ASSERT_TRUE(zmFloor.sourceSpace());
+  EXPECT_EQ(space34_.get(), zmFloor.sourceSpace().get());
+  EXPECT_FALSE(zmFloor.sourceZone());
+
+  ASSERT_TRUE(zmPerson.designFlowRate());
+  EXPECT_EQ(equivm3s_Person, zmPerson.designFlowRate().get());
+  EXPECT_FALSE(zmPerson.flowRateperFloorArea());
+  EXPECT_FALSE(zmPerson.flowRateperPerson());
+  EXPECT_FALSE(zmPerson.airChangesperHour());
+  ASSERT_TRUE(zmPerson.space());
+  EXPECT_EQ(space34_.get(), zmPerson.space().get());
+  EXPECT_FALSE(zmPerson.zone());
+  ASSERT_TRUE(zmPerson.sourceSpace());
+  EXPECT_EQ(space12_.get(), zmPerson.sourceSpace().get());
+  EXPECT_FALSE(zmPerson.sourceZone());
+
+  ASSERT_TRUE(zmACH.designFlowRate());
+  EXPECT_EQ(equivm3s_ACH, zmACH.designFlowRate().get());
+  EXPECT_FALSE(zmACH.flowRateperFloorArea());
+  EXPECT_FALSE(zmACH.flowRateperPerson());
+  EXPECT_FALSE(zmACH.airChangesperHour());
+  ASSERT_TRUE(zmACH.space());
+  EXPECT_EQ(space12_.get(), zmACH.space().get());
+  EXPECT_FALSE(zmACH.zone());
+  ASSERT_TRUE(zmACH.sourceSpace());
+  EXPECT_EQ(space34_.get(), zmACH.sourceSpace().get());
+  EXPECT_FALSE(zmACH.sourceZone());
+}
```

Comment:
Detail ThermalZone::combineSpaces test, and the order in which things are done. Some of that is tested in FT, but not all. So I think I like having both tests